src/services/challenge-pull/index.ts

```diff
@@ -244,7 +243,7 @@ export default class ChallengePullService implements IChallengePullService {
     }
 
     // Can not find linked issue.
-    const issueNumber = findLinkedIssueNumber(pullQuery.body);
+    const issueNumber = findLinkedIssueNumber(pullQuery.body || "");
```

Comment:
Why we need this empty string?

Reply:
The body can be null, so I patch it.

Reply:
I don't think this is the right way to handle null values. Maybe we should judge it in the function instead of passing the empty string to it.